app/controller/settings.js

```diff
@@ -0,0 +1,33 @@
+var fs = require('fs');
+var express = require('express');
+
+var router = express.Router();
```

Comment:
Yes you create a new object for the config which has the methods on it then use it in the routes file